Makefile

```diff
@@ -82,6 +82,7 @@ $(OUT)/trivialDAO_storage.tz: src/**
       src/base_DAO.mligo base_DAO_contract "default_full_storage( \
         { storage_data = \
           { admin = (\"$(admin_address)\" : address) \
+          ; guardian = (\"$(admin_address)\" : address) \
```

Comment:
guardian is always same as admin address? The acceptance description seems to describe otherwise,

>this cannot be held by an implicit account (to be checked in drop_proposal)
>its permissions are different from the admins, it's duties are only to call drop_proposal where necessary

So it appears that we need to add another argument for the user to specify guardian.

Reply:
> guardian is always same as admin address

It's not clear from the description, as it is _possible_ for the admin to be the guardian as well, but in most cases the two _should_ be different.

Note: the `guardian` is supposed to be a contract, so let's provide a `KT...` default value.

Reply:
Guardian address being admin address was just a placeholder since I was note quite understand it.

Now I've updated to be a contract address, and add an argument for it as well.